web/src/tippyjs.js

```diff
@@ -27,6 +27,35 @@ function get_tooltip_content(reference) {
     return "";
 }
 
+// We need to store instances together to destroy them in case of re-rendering.
+let tippyInstances = [];
```

Comment:
We don't need to track all tippy instances here, since we are only trying to fix this tooltips in message list. So, let's create something similar to `popver_menus.tippy_no_propagation` that we have for this:

```js

function tippy_no_propagation(target, props) {
        delegate("body", {
            target,
            appendTo: () => document.body, // Since the bug is only present in these tooltips 
            plugins: [globalStore],
            ...props,
        });
}
```

and use it instead of `delegate` for tooltips that are present in a message list.

Reply:
Sounds great, I have changed the code and logic accordingly